packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -993,6 +1044,11 @@ export default class CodeSubmode extends Component<Signature> {
       .accordion-content {
         padding: var(--boxel-sp-sm);
       }
+
+      pre.error {
+        white-space: pre-wrap;
+        text-align: left;
```

Comment:
I changed it to be left-aligned because centred looks weird:

<img width="572" alt="Boxel 2024-01-26 13-08-15" src="https://github.com/cardstack/boxel/assets/43280/c7027b06-8d7a-4363-8391-134869e05bda">

Arguably so does left-aligned, I think it would be better with the “Card preview failed” etc text left-aligned too but that seems like too much of a deviation…?

<img width="572" alt="Boxel 2024-01-26 13-07-26" src="https://github.com/cardstack/boxel/assets/43280/7924b497-5bd7-45cd-b37e-2a9a3092fedf">

Reply:
I suggest reusing our module errors display and dropping the generic failure message above, something like this:

<img width="920" alt="image" src="https://github.com/cardstack/boxel/assets/273660/3dae9aff-a358-43eb-8343-8e9428d41510">

Reply:
👍🏻 I’ll try that, thanks, I’m never sure when big departures from the design are warranted 😬

Reply:
I kept the error text because it might provide a clue when people are confused but I think this is better, thanks!

<img width="576" alt="Boxel 2024-01-29 15-37-09" src="https://github.com/cardstack/boxel/assets/43280/4d4775fa-b45e-49c4-a346-5843ac4db378">

Reply:
Nice!